docs/sources/next/javascript-api/k6-experimental/browser/request/timing.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'timing()'
+description: 'Browser module: Request.timing method'
+---
+
+# timing()
+
+Returns resource timing information for given request. Most of the timing values become available upon the response, `responseEnd` becomes available when request finishes.
```

Comment:
"for a given request" or "for the given request"?

Reply:
I've used "the" in 6f2167dc8fe1fbbc947727c0bacc3cef6ed3b478